arm-web/2016-03-01/swagger/DeletedWebApps.json

```diff
@@ -766,255 +766,244 @@
+          }
+        },
+        "machineKey": {
+          "$ref": "#/definitions/SiteMachineKey",
+          "description": "Site MachineKey.",
+          "readOnly": true
+        },
+        "handlerMappings": {
+          "description": "Handler mappings.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HandlerMapping"
+          }
+        },
+        "documentRoot": {
+          "description": "Document root.",
+          "type": "string"
+        },
+        "scmType": {
+          "description": "SCM type.",
+          "enum": [
+            "None",
+            "Dropbox",
+            "Tfs",
+            "LocalGit",
+            "GitHub",
+            "CodePlexGit",
+            "CodePlexHg",
+            "BitbucketGit",
+            "BitbucketHg",
+            "ExternalGit",
+            "ExternalHg",
+            "OneDrive",
+            "VSO"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ScmType",
+            "modelAsString": true
+          }
+        },
+        "use32BitWorkerProcess": {
+          "description": "<code>true</code> to use 32-bit worker process; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "webSocketsEnabled": {
+          "description": "<code>true</code> if WebSocket is enabled; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "alwaysOn": {
+          "description": "<code>true</code> if Always On is enabled; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "javaVersion": {
+          "description": "Java version.",
+          "type": "string"
+        },
+        "javaContainer": {
+          "description": "Java container.",
+          "type": "string"
+        },
+        "javaContainerVersion": {
+          "description": "Java container version.",
+          "type": "string"
+        },
+        "appCommandLine": {
+          "description": "App command line to launch.",
+          "type": "string"
+        },
+        "managedPipelineMode": {
+          "description": "Managed pipeline mode.",
+          "enum": [
+            "Integrated",
+            "Classic"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagedPipelineMode",
+            "modelAsString": false
+          }
+        },
+        "virtualApplications": {
+          "description": "Virtual applications.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VirtualApplication"
+          }
+        },
+        "loadBalancing": {
+          "description": "Site load balancing.",
+          "enum": [
+            "WeightedRoundRobin",
+            "LeastRequests",
+            "LeastResponseTime",
+            "WeightedTotalTraffic",
+            "RequestHash"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SiteLoadBalancing",
```

Comment:
Setting modelAsString to false can result in breaking changes whenever you modify this enum